DataFormats/SiStripDetId/interface/SiStripDetId.h

```diff
@@ -112,18 +112,18 @@ class SiStripDetId : public DetId {
 // ---------- inline methods ----------
```

Comment:
Thanks, I will rewrite the function.

Do I need the inline expression?
Or rather I do not understand why there is the comment

     // ---------- inline methods ----------
 
but no inline expressions, in the original code